board/overlay/external.desc

```diff
@@ -0,0 +1 @@
+name: OVM_EXTERNAL_OVERLAY
```

Comment:
```suggestion
name: OVM_EXTERNAL_OVERLAY

```

Reply:
to see https://buildroot.org/downloads/manual/manual.html#outside-br-custom.